recipes/requests-html/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "requests-html" %}
+{% set version = "0.10.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/requests-html-{{ version }}.tar.gz
+  sha256: 7e929ecfed95fb1d0994bb368295d6d7c4d06b03fcb900c33d7d0b17e6003947
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - bs4
+    - fake-useragent
+    - parse
+    - pyppeteer >=0.0.14
+    - pyquery
+    - python >=3.6
+    - requests
+    - w3lib
+
+test:
+  imports:
+    - requests_html
+  commands:
+    #- pip check #requests-html 0.10.0 requires bs4, which is not installed.
```

Comment:
It should be installed as it's in `run`?

Reply:
yes, it should and it looks like it does but it still produces that error for some reason.

Reply:
I don't see a failing commit, can you push one so the logs can be checked

Reply:
ok, i just edited and pushed

Reply:
@chrisburr the log shows the [error now](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=342776&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=1189)

Reply:
It's a bug in the package, `bs4` is the import but the package name is `beautifulsoup4`: https://github.com/psf/requests-html/pull/296

There is also an error from pyppeteer but that should be fixed by https://github.com/conda-forge/pyppeteer-feedstock/pull/9

Can you disable the check and add a comment that `pip check` should be re-enabled when there is a new release?